docs/target.settings.yaml

```diff
@@ -4,7 +4,18 @@ general:
   cm_device_codename: grouper
   cm_release_type: snapshot
   cm_release_version: M12
-  parent_template: mia-default
+  parent_templates:
+    - mia-default
+    - wifi-only
```

Comment:
yes, and any time you would add a new extra template, it will be appended to the list.